drivers/usb/device/usb_dc_mcux.c

```diff
@@ -34,6 +38,12 @@ static void usb_isr_handler(void);
 #define SETUP_DATA_STAGE_DONE	(0)
 #define SETUP_DATA_STAGE_IN	(1)
 #define SETUP_DATA_STAGE_OUT	(2)
+#define USBD_MCUX_THREAD_STACK_SIZE	(1024)
+
+/* endpoint related macros */
+#define EP0_MAX_PACKET_SIZE	(64)
+#define EP0_OUT			(0)
```

Comment:
Use USB_CONTROL_EP_IN  and USB_CONTROL_EP_IN instead.

Reply:
Thanks for pointing this, I have made the change.